sections/dashboard/FuturesMarketsTable/FuturesMarketsTable.tsx

```diff
@@ -19,6 +19,7 @@ import useGetAverageFundingRateForMarkets, {
 	FundingRateResponse,
 } from 'queries/futures/useGetAverageFundingRateForMarkets';
 import { Period, PERIOD_IN_SECONDS } from 'constants/period';
+import { DEFAULT_FIAT_EURO_DECIMALS } from '../../../constants/defaults';
```

Comment:
Good catch 👍 ! VS code auto import isn't smart enough sometimes so I had to update the imports by myself. Since I  haven't read through the change myself this got missed.